src/js/categories/render-template.js

```diff
@@ -0,0 +1,12 @@
+// render-template.js
+export const createCard = ({ filter, name, imgURL }) => `
+<li>
```

Comment:
card should be a link with query parameters, so you can click and fetch data by filter and name